packages/jyve-pyodide-unsafe/src/kernel.ts

```diff
@@ -110,8 +110,15 @@ export namespace PyodideUnsafeKernel {
     bootstrap_pyodide(window);
 
     let timeout = 0.5;
-    while (!(window.pyodide && window.pyodide.runPython)) {
+    let count = 30 / timeout;
+    while (!(window.pyodide && window.pyodide.runPython) && count > 0) {
       await JyveKernel.wait(timeout);
+      count--;
+    }
+
+    if (count <= 0) {
+      // TODO: Is there a way to popup an error modal here?
+      alert("Pyodide failed to load.");
```

Comment:
This is fine for now... cracking open a dialog is [pretty easy](https://github.com/jupyterlab/jupyterlab/blob/afe47dde9bb1c181f7d546a1f696d02d531f3114/packages/application-extension/src/index.tsx#L81).

However, I am wondering if it wouldn't be more appropriate to show an in-notebook error, but I don't think there are any examples of that yet in jyve. I'd have to look at what a real kernel would do...